km_api/know_me/tests/serializers/test_km_user_accessor_serializer.py

```diff
@@ -0,0 +1,31 @@
+from know_me import serializers
+
+
+def test_serialize(
+        api_rf,
+        km_user_factory,
+        km_user_accessor_factory,
+        serializer_context,
+        user_factory):
+    """
+    Test serializing a km_user_accessor.
+    """
+    km_user = km_user_factory()
+    user = user_factory()
+    km_user_accessor = km_user_accessor_factory(
+        km_user=km_user,
+        user_with_access=user)
+
+    serializer = serializers.KMUserAccessorSerializer(
+        km_user_accessor,
+        context=serializer_context)
+
+    expected = {
+        'accepted': False,
+        'can_write': False,
+        'has_private_profile_access': False,
+        'km_user': km_user,
+        'user_with_access': user,
```

Comment:
Users have no access to other user's account information, so this field doesn't make sense to include. My suggestion is to replace this field with the email address used to invite the user.

Reply:
Instead of having the `user_with_access` field we should have a `user_with_access_email` field in the serializer?

Reply:
That could work. If you have a better name for it feel free to use it.